YellowNotes.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,83 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using YellowNotes.Core.Dtos;
+using YellowNotes.Core.Services;
+using YellowNotes.Core.Utility;
+using System.Threading;
+
+namespace YellowNotes.Api.Controllers
+{
+    [Authorize]
```

Comment:
That should be moved to kind of global things because that's our approach

Reply:
You mean `GlobalFilters`?